### PR TITLE
Renames sequence events to actions

### PR DESCRIPTION
**Pull request by mitschabaude**
companion of https://github.com/MinaProtocol/mina/pull/12367

### CODE REVIEW

File: src/lib/account_update.ts [hunk 1]
```diff
@@ -360,7 +360,7 @@ interface Body extends AccountUpdateBody {
    *
    * [Check out our documentation about Actions!](https://docs.minaprotocol.com/zkapps/advanced-snarkyjs/actions-and-reducer)
    */
-  sequenceEvents: Events;
+  actions: Events;
```
Comment (mitschabaude) [inline src/lib/account_update.ts:363]: would be nice to change the comment above this!

File: src/lib/account_update.ts [hunk 2]
```diff
@@ -585,14 +585,14 @@ class AccountUpdate implements Types.AccountUpdate {
   network: Precondition.Network;
   children: {
     callsType:
-      | { type: 'None' }
-      | { type: 'Witness' }
-      | { type: 'Equals'; value: Field };
+    | { type: 'None' }
```
Comment (mitschabaude) [inline src/lib/account_update.ts:588]: we should revert all this autoformatting

**Review by mitschabaude — COMMENTED** [review]
LGTM with some comments (someone else needs to approve since I opened this PR)

EDIT: However, I think there is a `SequenceEvents` module / type on the snarkyjs side that we should still rename

**Review by Trivo25 — APPROVED** [review]

**Comment by bkase** [comment]
I figured the other SequenceEvents name internally could be handled in another PR

File: src/lib/zkapp.ts [hunk 3]
```diff
@@ -1306,7 +1306,16 @@ Use the optional \`maxTransactionsWithActions\` argument to increase this number
               )
             )
           );
-      });
+      }
+
+      if (Circuit.inCheckedComputation()) {
```
Comment (Trivo25) [inline src/lib/zkapp.ts:1311]: A recent update in the Mina repo changed how snarky behaves. It used to set `inCheckedComputation = true` after *any* contract compilation - that now changed for some reason

File: src/lib/int.ts [hunk 4]
```diff
@@ -108,7 +108,7 @@ class UInt64 extends CircuitValue {
       };
     }
 
-    y_ = y_.seal();
+    y_ = y_.seal(); //Circuit.inCheckedComputation() ? y_.seal() : y_;
```
Comment (bkase) [inline src/lib/int.ts:111]: @Trivo25 is it important we keep this comment?

**Review by bkase — APPROVED** [review]
Looks good!

**Pull request merged**